.github/buildomat/jobs/package.sh

```diff
@@ -42,17 +43,17 @@ tarball_src_dir="$(pwd)/out"
 files=(
 	out/*.tar
 	package-manifest.toml
-	smf/sled-agent/config.toml
+	smf/sled-agent/nongimlet/config.toml
```

Comment:
This only includes the `nongimlet` config file now.
Does that mean we don't need `smf/sled-agent/config.toml`, or that this build will only be for nongimlet systems?

Reply:
This *particular* package is only being used by the `deploy` buildomat job, which can only run on non-gimlet systems.

Separately, we actually are building the `smf/sled-agent/gimlet/config.toml` into a tarball that will become part of the regular host OS image.